exporter/clickhouseexporter/config.go

```diff
@@ -71,15 +63,11 @@ func (cfg *Config) Validate() (err error) {
 		err = errors.Join(err, e)
 	}
 
-	return err
-}
-
-func (cfg *Config) enforcedQueueSettings() exporterhelper.QueueSettings {
-	return exporterhelper.QueueSettings{
-		Enabled:      true,
-		NumConsumers: 1,
-		QueueSize:    cfg.QueueSettings.QueueSize,
+	if e := cfg.QueueSettings.Validate(); e != nil {
+		err = errors.Join(err, e)
 	}
```

Comment:
This is not needed. `Validate()` is called for every embedded struct, see https://github.com/open-telemetry/opentelemetry-collector/blob/f0bba0ec3521fb726b17137d1c5672566aeb79f5/component/config.go#L50

Reply:
Ah thanks for this info.